content/sensu-go/5.16/api/assets.md

```diff
@@ -125,21 +155,20 @@ The `/assets/:asset` API endpoint provides HTTP GET access to [asset data][1] fo
 In the following example, querying the `/assets/:asset` API endpoint returns a JSON map that contains the requested [`:asset` definition][1] (in this example, for the `:asset` named `check_script`).
 
 {{< highlight shell >}}
-curl http://127.0.0.1:8080/api/core/v2/namespaces/default/assets/sensu-slack-handler -H "Authorization: Bearer $SENSU_TOKEN"
+curl -X GET http://127.0.0.1:8080/api/core/v2/namespaces/default/assets/sensu-slack-handler
```

Comment:
requires ' \ ' at the end of the line, probably should move 'curl -X GET' to it's own line for consistency